src/main/java/net/mcreator/ui/minecraft/boundingboxes/JBoundingBoxList.java

```diff
@@ -53,7 +51,8 @@ public JBoundingBoxList(MCreator mcreator) {
 		add("Center", new JScrollPane(PanelUtils.pullElementUp(entries)));
 
 		add.addActionListener(e -> {
-			registerEntryUI(new JBoundingBoxEntry(entries, boundingBoxList).setEntryEnabled(this.isEnabled()));
+			JBoundingBoxEntry entry = new JBoundingBoxEntry(entries, boundingBoxList).setEntryEnabled(this.isEnabled());
+			registerEntryUI(entry);

```

Comment:
Why did you split this line? Is it more atomic now?

Reply:
So JBoundingBoxEntry is added to the entry list before registering UI

Reply:
It should be the case before too, but this is more readable for someone coming to this codebase

Reply:
I don't think it changes anything. Both before and now entry is first initialized, then registered.

Reply:
Indeed, as I said it should be the case before too but should be more readable for new users, as it can be confusing to initiate the object in registerEntryUI directly.

Reply:
Ah, right.